index.bs

```diff
@@ -1527,6 +1531,483 @@ longer be considered as sanitized.
 
 A more complete treatment of mXSS can be found in [[MXSS]].
 
+
+# Index
+
+<em>This section is not normative.</em>
+
+## Elements and attributes not included in the default config ##  {#default-disallowed-elements}
+
+While the Sanitizer aims to disallow Cross-Site Scripting attacks, the default
+settings are a bit stricter and do not include elements that may change page
+settings, layout or fetch resources from other origins. The following elements
+can be allowed with a configuration, but the individual reasoning for disallowing
+them by default is given below.
```

Comment:
This list seems very incomplete. I think it either needs to cover all elements and attributes we made a decision on or we should not have it.

Reply:
+1. This was my first reaction after looking at this list as well.